config/auth_azure_cli.go

```diff
@@ -82,7 +82,7 @@ func (ts *azureCliTokenSource) Token() (*oauth2.Token, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot parse expiry: %w", err)
 	}
-	logger.Infof("Refreshed OAuth token for %s from Azure CLI, which expires on %s",
+	logger.Infof(context.Background(), "Refreshed OAuth token for %s from Azure CLI, which expires on %s",
```

Comment:
@nfx Ideally we have access to a context here as well but `oauth2.TokenSource` doesn't deal with contexts...